open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-basics.ipynb

```diff
@@ -1,12 +1,27 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        def test_string_replaced(self):
</pre><p>This is just the happy case. We should have unhappy case later. e.g. to test an empty string, or a string has nothing to replace</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>